package.json

```diff
@@ -74,6 +75,7 @@
     "@types/node": "^24.9.2",
     "@types/yargs": "^17.0.34",
     "@typescript/native-preview": "^7.0.0-dev.20251027.1",
+    "husky": "^9.1.7",
```

Comment:
Not sure how this got in, this repo should use `simple-git-hooks` instead - but it used husky before and maybe something is causing problems. I'll look into it